### PR TITLE
Add new spec and parser for "ls -lan /usr/bin"

### PR DESCRIPTION
**Pull request by JoySnow**
Signed-off-by: XiaoXue Wang <xiaoxwan@redhat.com>

### CODE REVIEW

**Review by xiangce — APPROVED** [review]
Approved pending approval of Spec.

**Review by bfahr — APPROVED** [review]

**Pull request merged**
